public/js/widget-manager.js

```diff
@@ -34,10 +35,24 @@ define(['jupyter-js-widgets'], function(Widgets) {
     };
     WidgetManager.prototype = Object.create(Widgets.ManagerBase.prototype);
 
+    /*
+     * Called when a ipywidget is added to the DOM.
```

Comment:
Yes. That's true. The JS side is jupyter widget. The backend side is still an ipywidget (for now).